share/spice/open_nic/open_nic.js

```diff
@@ -0,0 +1,28 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_open_nic = function(api_result){
+
+        if (!api_result) {
+            return Spice.failed('open_nic');
+        }
+
+        Spice.add({
+            id: "open_nic",
+            name: "OpenNIC DNS",
```

Comment:
The tab name should probably just be "DNS" or "Answer" might even be better. We try to prevent proper nouns as tab names as much as possible.